client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -279,6 +280,17 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			} );
 		}
 
+		if ( EditorActions.GetGutenboardingStatus === action ) {
+			// TODO - In future iterations, replace window param with gutenboarding site info.
+			const urlParams = new URLSearchParams( window.location.search );
+			const isGutenboarding =
+				config.isEnabled( 'gutenboarding' ) && urlParams.has( 'is-gutenboarding' );
+			ports[ 0 ].postMessage( {
+				isGutenboarding,
+				frankenflowUrl: `${ window.location.origin }/start/frankenflow`,
```

Comment:
We're supposed to provide the `siteSlug` for frankenflow. Site ID is usually interchangeable and this seems to work:

```suggestion
				frankenflowUrl: `${ window.location.origin }/start/frankenflow?siteSlug=${ this.props.siteId }`,
```

Reply:
sounds good!